kubevirt_handler/metadata.csv

```diff
@@ -0,0 +1,47 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric,sample_tags
+kubevirt_handler.can_connect,gauge,,,,Whether the check can connect to the KubeVirt Handler or not.,0,kubevirt_handler,handler connect,,
```

Comment:
I would add 1 if able 0 if not or something along those lines

Reply:
I'll look into that in a follow up PR where I will align all the kubevirt integrations.